lib/active_merchant/billing/gateways/paypal_checkout.rb

```diff
@@ -0,0 +1,376 @@
+require 'active_merchant/billing/gateways/paypal/paypal_checkout_common'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class PaypalCheckoutGateway < Gateway
+      include PaypalCheckoutCommon
+      
+      self.supported_countries = ['AU', 'AT', 'BE', 'BG', 'CA', 'CY', 'CZ', 'DK', 'EE', 'FI', 'FR', 'GR', 'HU', 'IT', 'LV', 'LI', 'LT', 'LU', 'MT', 'NL', 'NO', 'PL', 'PT', 'RO', 'SK', 'SI', 'ES', 'SE', 'US', 'GB']

```

Comment:
do we use it somewhere?

Reply:
No @inai-shreyas, we are not using it but active-merchant exposes a method to check for the given `country_code` if it is present in this list/not. 
It is included as part of every PSPGateway in active-merchant so thought I'd add it here.